scripts/run_analysis_sim.py

```diff
@@ -27,13 +27,21 @@ def run_analysis_sim(num_of_robots):
         for j in range(ary_map.shape[1]):
             px = sum(ary_map[i, j])
             if px == 0:
-                shelves.add((i, j))
-                if len(goals) < num_of_robots:
-                    goals.append((i, j, 0))
+                obstacles.add((i, j))
```

Comment:
this should also add obstacles if px=255, right?

Reply:
No, not in this revision at least. It was causing problems when the orchestrator tried to path the robots back to the charging station since I was treating it as homebase.